### PR TITLE
feat: add a switch of access log

### PR DESCRIPTION
**Pull request by HelloBug0**
### What this PR does / why we need it:
satify the need of [#2713](https://github.com/apache/apisix/issues/2713)

### Pre-submission checklist:

* [X] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [X] Have you modified the corresponding document?
* [ ] Is this PR backward compatible?

### CODE REVIEW

File: conf/config-default.yaml [hunk 1]
```diff
@@ -128,6 +128,7 @@ nginx_config:                     # config for render the template to genarate n
   #envs:                            # allow to get a list of environment variables
   #  - TEST_ENV
   http:
+    access_log_switch: on          # on or off, default on
```
Comment (spacewander) [inline conf/config-default.yaml:131]: Better to use `enable_access_log: true` to ensure consistency.

**Comment by spacewander** [comment]
#2765 should be put into this one.

File: conf/config-default.yaml (hunk 1 above)
Comment (HelloBug0) [reply conf/config-default.yaml:131]: yeah, right

**Comment by HelloBug0** [comment]
> #2765 should be put into this one.

It occurs to me, but it's kind of awkward that I still don't know how to PR once with two changed files in windows even after I searched that in the Internet. I realize that I need to learn more about git operations.

**Pull request closed without merging**